game/lib/main.dart

```diff
@@ -71,7 +73,7 @@ class MyApp extends StatelessWidget {
           ],
           supportedLocales: const [Locale('en', '')],
           theme: ThemeData(primarySwatch: Colors.blue),
-          home: SplashPageWidget(),
```

Comment:
Keep this to the registration page we don't want a page to take over the master branch. I will make a way to navigate here later.